src/Dockerfile-base

```diff
@@ -35,6 +35,8 @@ RUN conda install -y python=${python_version} scikit-learn six h5py \
         matplotlib pillow rasterio jupyter scikit-image && \
     conda clean -yt
 
+RUN pip install spectral
```

Comment:
When you add a dependency, it's a good idea to say in the commit message why that dependency was added.